recipes/quaternions/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package_multi"
```

Comment:
I added the `cmake_find_package_multi` generator here, because the `cmake` generator did not accept targets like `PackageName::PackageName` only in the form of `CONAN_PKG::PackageName`